.github/workflows/master.yml

```diff
@@ -0,0 +1,91 @@
+name: Master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout changes"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
```

Comment:
nit: do these jobs need a `fetch-depth` of 2? Doesn't look like we're working with anything but the current commit

Reply:
🤷‍♂️  probably a relic from copying the code from somewhere else. let me try with just pulling the last commit.